CHANGES.md

```diff
@@ -1,6 +1,28 @@
 # Changelog
 
-## 1.12.3 (core) / 0.28.3 (libraries)
+## 1.12.4 (core) / 0.28.4 (libraries)
+
+### New
+
+- CI workflows for Gitlab projects can now be scaffolded using `dg plus deploy configure`.
+- "/" characters are now allowed in concurrency pool names.
+- Pod wait timeout for K8sPipeClient can now be specified (Thanks, [@abhinavDhulipala](https://github.com/abhinavDhulipala)!)
+- New `kind` tag icon for Zendesk (Thanks, [@kporter13](https://github.com/kporter13)!)
+
+### Bugfixes
+
+- Fixed an issue where passing in JSON serializable enums to JsonMetadataValue would sometimes result in an error.
+- Fixed an issue that would cause `SensorDefinition` subclasses (e.g. `AutomationConditionSensorDefinition`, `RunStatusSensorDefinition`) to be converted to having the wrong `sensor_type` property when produced from a `Component`.
+- feat: tableau-component: datasource_refresh and fixes.
```

Comment:
https://github.com/dagster-io/dagster/pull/32829 is the associated PR - i need to read through it and parse the actual changes

Reply:
ok i was able to parse the bug fixes, but i'm not sure what the entries for the new features in this PR should be - @OwenKephart could you help out?